unified-runtime/source/common/ur_pool_manager.hpp

```diff
@@ -63,6 +63,10 @@ struct pool_descriptor {
                                   const pool_descriptor &desc);
   static std::pair<ur_result_t, std::vector<pool_descriptor>>
   create(ur_usm_pool_handle_t poolHandle, ur_context_handle_t hContext);
+  static std::pair<ur_result_t, std::vector<pool_descriptor>>
+  createFromDevices(ur_usm_pool_handle_t poolHandle,
+                    ur_context_handle_t hContext,
+                    std::vector<ur_device_handle_t> devices);
```

Comment:
nit: you can pass the vector as `const &`

Reply:
Done.